### PR TITLE
Added ProactiveEvents API

### PR DESCRIPTION
**Pull request by omenocal**
Official documentation here:
https://developer.amazon.com/docs/smapi/proactive-events-api.html

Latest Alexa SDK model package updated to 1.11.1, no types for ProactiveEvents objects

### CODE REVIEW

File: docs/alexa-apis.rst [hunk 1]
```diff
@@ -817,25 +801,22 @@ In the following example, you'll see a simple code of a lambda function which ca
...
-          const request = {
-            endpoint: user.endpoint,
-            userId: user.userId,
-            data,
-            skillMessagingToken: result.access_token,
-          };
-
-          return Messaging.sendMessage(request);
-        })
+      const data = {
```
Comment (armonge) [inline docs/alexa-apis.rst]: what's up with this changes related to the messaging api?

File: src/platforms/alexa/apis/ProactiveEvents.ts [hunk 2]
```diff
@@ -0,0 +1,576 @@
...
+export enum OCCASION_CONFIRMATION_STATUS {
+  CANCELED = "CANCELED",
+  CONFIRMED = "CONFIRMED",
+  CREATED = "CREATED",
+  REQUESTED = "REQUESTED",
+  RESCHEDULED = "RESCHEDULED",
+  UPDATED = "UPDATED",
+}
+
+export enum OCCASION_TYPE {
```
Comment (armonge) [inline src/platforms/alexa/apis/ProactiveEvents.ts]: could we take this directly from ask-sdk-model?

File: src/platforms/alexa/apis/index.ts [hunk 3]
```diff
@@ -4,4 +4,30 @@ export { DeviceSettings } from "./DeviceSettings";
 export { InSkillPurchase } from "./InSkillPurchase";
 export { Lists } from "./Lists";
 export { IMessageRequest, Messaging } from "./Messaging";
+export {
+  EventsBuilder,
```
Comment (armonge) [inline src/platforms/alexa/apis/index.ts]: maybe we should look into typescript namespaces?

File: src/platforms/alexa/apis/ProactiveEvents.ts (hunk 2 above)
Comment (omenocal) [reply src/platforms/alexa/apis/ProactiveEvents.ts]: In the PR's description I mentioned, the ask-sdk-model package in its latest version does not have types for this yet. I did see, they have included the types for Reminders API, so I will implement those changes in my next PR

File: src/platforms/alexa/apis/index.ts (hunk 3 above)
Comment (omenocal) [reply src/platforms/alexa/apis/index.ts]: Same as my last comment

**Pull request closed without merging**